File: benchmark/compare/standard/string_interp.lua
local start = os.clock()
local result = ""
for i = 0, 9999 do
    result = string.format("item_%d", i)
end
local elapsed = (os.clock() - start) * 1000
print(string.format("last = %s", result))
print(string.format("elapsed_ms=%.2f", elapsed))

File: benchmark/compare/standard/string_interp.js
let start = performance.now();
let result = "";
for (let i = 0; i < 10000; i++) { result = `item_${i}`; }
let elapsed = performance.now() - start;
console.log(`last = ${result}`);
console.log(`elapsed_ms=${elapsed.toFixed(2)}`);
